modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java

```diff
@@ -52,6 +51,42 @@
 @Deprecated
 public class AssetTagsSelectorTag extends IncludeTag {
 
+	public String getAddCallback() {
+		return _addCallback;
+	}
+
+	public String getClassName() {
+		return _className;
+	}
+
+	public long getClassPK() {
+		return _classPK;
+	}
+
+	public String getHiddenInput() {
+		return _hiddenInput;
+	}
+
+	public String getRemoveCallback() {
+		return _removeCallback;
+	}
+
+	public boolean isAllowAddEntry() {
```

Comment:
Methods need to be alphabetically sorted (run ant format-source from portal-impl/ directory).

Reply:
They're sorted, aren't them?